src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,84 @@
+$days-in-a-row: 7;
+
+$cell-width: 100px;
+$cell-height: 100px;
+
+%cell-size {
+  width: $cell-width;
+  height: $cell-height;
+}
+
+$calendar-inner-gap: 1px;
+$calendar-padding-x: 10px;
+$calendar-width: $days-in-a-row * $cell-width +
+  $calendar-inner-gap * ($days-in-a-row - 1);
+
+$week-days: ("mon": 0, "tue": 1, "wed": 2, "thu": 3,
+"fri": 4, "sat": 5, "sun": 6);
+
+@function first-day-ident($day-number) {
+  @return $day-number * ($cell-width + $calendar-inner-gap);
+}
+
```

Comment:
you can make another files for variables, placeholders, functions and mixins, if u want of course..